probeep.yaml

```
@@ -69,7 +82,29 @@ spec:
           periodSeconds: 5
           timeoutSeconds: 1
       imagePullSecrets:
-      - name: registry
+      - name: lucca-registry
       restartPolicy: Always
       terminationGracePeriodSeconds: 20
       serviceAccountName: probeep
+---
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: consul-ep
+  annotations:
+    lucca.net/probe-ep-hostnames: '["http://hostname.to.resolve.example.com:8500","http://hostname.to.resolve2.example.com:8500","http://hostname.to.resolve3.example.com:8500"]'
```

Maybe we can use this tool with multiple protocols ?
I think the port number isn't used ? If so, can we remove it ?

Can we clean these targets and give only hostnames ?

```suggestion
    lucca.net/probe-ep-hostnames: '["hostname.to.resolve.example.com","hostname.to.resolve2.example.com","hostname.to.resolve3.example.com"]'
```

---

Yes the protocol and the port from the annotation aren't used.
Removing it.